From 6050f8ed7109886a042c749f7fbb2858051075e4 Mon Sep 17 00:00:00 2001
From: Ayushi Smriti <smriti.ayushi@intel.com>
Date: Wed, 23 Sep 2020 22:01:25 +0530
Subject: [PATCH] Add PLDM version purpose enumeration

This change is to add PLDM in enumeration of possible purposes
of the version to support pldm type version purpose

Change-Id: I7b914d4323bfe44a4e3cd60ed4a627aeceb6b56f
Signed-off-by: Ayushi Smriti <smriti.ayushi@intel.com>
---
 yaml/xyz/openbmc_project/Software/Version.interface.yaml | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/yaml/xyz/openbmc_project/Software/Version.interface.yaml b/yaml/xyz/openbmc_project/Software/Version.interface.yaml
index 6176010..a2b1eb6 100644
--- a/yaml/xyz/openbmc_project/Software/Version.interface.yaml
+++ b/yaml/xyz/openbmc_project/Software/Version.interface.yaml
@@ -41,3 +41,6 @@ enumerations:
           - name: PSU
             description: >
                 The version is a version for a PSU.
+          - name: PLDM
+            description: >
+                The version is a version for a PLDM.
